src/meta/src/manager/id.rs

```diff
@@ -175,6 +177,7 @@ where
                 )
                 .await,
             ),
+            sink: Arc::new(StoredIdGenerator::new(meta_store.clone(), "sink", None).await),
```

Comment:
Since `create_sink` is using `Table` category and reusing code of creating materialized view, the fragment graph will be stored as `TableFragment` in `FragmentManager`. The key in manager is `SinkId` and `TableId`.  That's acceptable and this sink category is never used and can be removed.